### PR TITLE
Adding contributing guide pointer markdown

### PR DESCRIPTION
**Pull request by OBWANDO**
Signed-off-by: Royal OBrien <obwando@obwando.com>

### CODE REVIEW

**Review by sptramer — APPROVED** [review]

**Review by AMZN-alexpete — COMMENTED** [review]
Which is the source of truth - [the docs](https://o3de.org/docs/contributing/) or the [community repo](https://github.com/o3de/community/blob/main/CONTRIBUTING.md)?  Keeping multiple locations up to date is a risk especially since they're all in separate repositories and the content in these locations already appears to be different.

**Comment by sptramer** [comment]
@AMZN-alexpete I have this concern as well, especially for docs. Right now I think it's important to get this merged so we have _some_ `CONTRIBUTING.md` but we should revisit and make sure that the contributing information lives in a centralized location. We may want to move it off the website, or restructure the way the contributor guide is presented.

**Comment by AMZN-alexpete** [comment]
I'd like to point out that the contributing guide we're pointing them to has the following issues:
- The O3DE Contributor Guide link is broken http://github.com/o3de/community/contributors/guide
- The cheat sheet link is broken https://github.com/o3de/community/blob/main/github.com/o3de/community/contributors/guide/contributor-cheatsheet
- The contributing code section only mentions DCO and the license, it does not address workflow or validation like https://o3de.org/docs/contributing/to-code/ does
If we're going to send users to https://github.com/o3de/community/blob/main/CONTRIBUTING.md lets at least make sure all the essential links work first

**Review by AMZN-alexpete — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+Welcome to the Open 3D Engine! To learn more about contributing to the [O3DE code repo](README.md), check out the [Contributor's Guide](https://github.com/o3de/community/blob/main/CONTRIBUTING.md).
```
Comment (amzn-changml) [inline CONTRIBUTING.md:3]: This link is pointed to the main branch, but the PR is going into development?

**Comment by amzn-changml** [comment]
> @AMZN-alexpete I have this concern as well, especially for docs. Right now I think it's important to get this merged so we have _some_ `CONTRIBUTING.md` but we should revisit and make sure that the contributing information lives in a centralized location. We may want to move it off the website, or restructure the way the contributor guide is presented.

That's fair, we can bypass checks for these sorts of cases. The integrators team can do this once we get consensus

File: CONTRIBUTING.md (hunk 1 above)
Comment (AMZN-alexpete) [reply CONTRIBUTING.md:3]: I believe the community repository only has a main branch

**Pull request merged**